taker-electron/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+use neon::prelude::*;
+use once_cell::sync::OnceCell;
+use taker::Opts;
+use tokio::runtime::Runtime;
+
+// Return a global tokio runtime or create one if it doesn't exist.
+// Throws a JavaScript exception if the `Runtime` fails to create.
```

Comment:
🔧 You probably want these comments to be docstrings?

```suggestion
/// Return a global tokio runtime or create one if it doesn't exist.
/// Throws a JavaScript exception if the `Runtime` fails to create.
```